langchain/src/agents/tools/serpapi.ts

```diff
@@ -94,4 +74,17 @@ export class SerpAPI extends Tool {
 
   description =
     "a search engine. useful for when you need to answer questions about current events. input should be a search query.";
+
+  static async imports(): Promise<{
```

Comment:
The new pattern I've settled on for importing peer dependencies, thoughts @sullivan-sean ?